agent/sshd/cmd_docker.go

```diff
@@ -12,10 +12,7 @@ import (
 )
 
 func newCmd(u *osauth.User, shell, term, host string, command ...string) *exec.Cmd {
-	uid, _ := strconv.Atoi(u.UID)
-	gid, _ := strconv.Atoi(u.GID)
-
-	nscommand, _ := nsenterCommandWrapper(uid, gid, fmt.Sprintf("/host/%s", u.HomeDir), command...)
+	nscommand, _ := nsenterCommandWrapper(u.UID, u.GID, fmt.Sprintf("/host/%s", u.HomeDir), command...)
```

Comment:
here as well.

Reply:
Doesn't this commit also belong to the previous one?

Reply:
squashed =D